### ISSUE TITLE
Other player icons flash with my icon

### ISSUE THREAD

**Issue by JCharante** [baxter-oop/AndOre#31]
Example:

```
!...............................
....+...........+..............+
...........+....................
..........................+.....
.................$..............
................................
................................
.............@.................+
..............$.................
..............$..............$..
+$......$.............+.........
.+..............................
................................
..........+.....................
.$........$......+..............
.$...........$......+......+....
................................
..............+.................
...........................!....
......................$...+.....
...........................$...$
...............................$
..+....$..$.....................
..............$.................
..............+.................
....$........................+..
...........+....................
.................$............$.
..................+.............
................................
................................
[hp 32 ore 33] [7 13] [][952]  
```
```
@...............................
....+...........+..............+
...........+....................
..........................+.....
.................$..............
................................
................................
.............@.................+
..............$.................
..............$..............$..
+$......$.............+.........
.+..............................
................................
..........+.....................
.$........$......+..............
.$...........$......+......+....
................................
..............+.................
...........................!....
......................$...+.....
...........................$...$
...............................$
..+....$..$.....................
..............$.................
..............+.................
....$........................+..
...........+....................
.................$............$.
..................+.............
................................
................................
[hp 32 ore 33] [7 13] [][952]  
```
```
!...............................
....+...........+..............+
...........+....................
..........................+.....
.................$..............
................................
................................
.............@.................+
..............$.................
..............$..............$..
+$......$.............+.........
.+..............................
................................
..........+.....................
.$........$......+..............
.$...........$......+......+....
................................
..............+.................
...........................!....
......................$...+.....
...........................$...$
...............................$
..+....$..$.....................
..............$.................
..............+.................
....$........................+..
...........+....................
.................$............$.
..................+.............
................................
................................
[hp 32 ore 33] [7 13] [][954]  
```

**Comment by JCharante** [comment]
Temporarily fixed in #35

**Comment by JCharante** [comment]
Closing the ticket as while it was temporarily fixed by re caching the rendered_world after every request, due to https://github.com/baxter-oop/AndOre/issues/21 the world is rendered many times per tick, making the temporary fix acceptable for now, but I will leave the console message in as a warning because I believe it useful to find out why the issue is/was happening.
